ios/FluentUI.Demo/FluentUI.Demo/ReadmeViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+
+class ReadmeViewController: UIViewController {
+
+    let scrollView = UIScrollView()
+    let contentView = UIView()
+
+    let readme: String?
+
+    let readmeLabel: Label = {
+        let label = Label()
+        label.numberOfLines = 0
+        label.sizeToFit()
```

Comment:
why do we need size to fit here in the inialization process?

Reply:
It's not needed. Went ahead and removed it.